src/classes/BDI_DataImportDeleteBTN_TEST.cls

```diff
@@ -41,6 +41,15 @@ class BDI_DataImportDeleteBTN_TEST {
     /** @description Error message returned when user does not have Delete permission on the SObject */
     static String INSUFFICIENT_ACCESS_ERROR_MSG = 'INSUFFICIENT_ACCESS_OR_READONLY';
 
+    /** @description Identifier for when Salesforce classic is displayed for the user */
```

Comment:
Can you reference these defined constants from the original controller (even if you have to make those test visible?)

Reply:
fixed